Dockerfile

```diff
@@ -4,9 +4,7 @@ FROM pytorch/torchserve:0.4.2-cpu as base
 USER root
 
 FROM base as reqs
-RUN pip3 install --upgrade pip
-RUN pip install cropharvest==0.3.0 google-cloud-storage netCDF4 pandas rasterio xarray
-RUN pip3 install torch==1.9.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+RUN pip3 install --disable-pip-version-check --no-compile pip -U torch==1.9.0+cpu -f https://download.pytorch.org/whl/torch_stable.html && pip install --disable-pip-version-check --no-compile cropharvest==0.3.0 google-cloud-storage netCDF4 pandas rasterio xarray
```

Comment:
The above command speeds up the pip installation command. The --disable-pip-version-check skips checking the pip version at the time of installation and --no-compile skips the compilation steps of the source files of the packages. The above steps have proved to reduce the build time and size of the docker image.

Reply:
How much do they reduce the build time and size of the docker image?

Reply:
I don't understand what you mean by previous version of image file and latest image file. Could you elaborate?

I read a bit about `--no-compile` option and I am not sure it necessarily speeds things up.

From https://pythonspeed.com/articles/faster-pip-installs/
> So should you always use this option? Not necessarily. Just because pip install is faster doesn’t mean you’ve saved time overall.
Any module you import will still need to be compiled into a .pyc, it’s just that the work will happen at Python run time, instead of at package installation time. So if you’re importing all or most modules, overall you might not save any time at all, you’ve just moved the work to a different place.

What do you think about the above @Aniket-Parlikar ?

> The size of the docker image has been reduced by around 30mb

Interesting! Is this due to the compiled pyc files not being there?

Reply:
Yes, we can close the PR